docs/sources/reference/components/loki/loki.process.md

```diff
@@ -1523,6 +1501,7 @@ stage.structured_metadata {
       env  = "",         // Sets up an 'env' property to structured metadata, based on the 'env' extracted value.
       user = "username", // Sets up a 'user' property to structured metadata, based on the 'username' extracted value.
     }
+    regex = "label_.*"   // Adds all extracted values starting with 'label_' to structured metadata.
```

Comment:
```suggestion
    regex = "__pod_.*"   // Adds all extracted values starting with '__pod_' to structured metadata.
```

A more realistic example would be nice.

Reply:
This is actually exactly the use case for which I was implementing the feature. I want to use a `discovery.relabel` component to rename all `__meta_kubernetes_pod_label_(.*)` to `label_$1` and then use `structured_metadata` in `loki.process` to move them to structured metadata.
But I see that the example `label_.*` is not very helpful because it is not clear that it refers to kubernetes pod labels instead of some generic label. But I think the `__pod_.*` example is not better because it is probably not best practice to add custom double-underscore metadata to Loki, or is it?